Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,134 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
                     </Flex>
                 </Flex>
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 1 Temp'
-                    data={props.data?.string1_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={1}
+                    overflowY='scroll'
+                >
+                    <GridItem colSpan={1}></GridItem>
+                    <div>Temp(ºC)</div>
+                    <div>Voltage in (V)</div>
+                    <div>Current in (A)</div>
+                    <GridItem fontSize={12}>String 1</GridItem>
+                    <GridItem colSpan={0.1}>
+                        <RangeCell
+                            fontSize='1vw'
```

Comment:
Using the  `vw` unit for font size apparently can break zooming, I'm not sure if this matters but percentages or fractions might make this more robust when the window size or zoom changes

Reply:
Although it's not an ideal solution, let's stick with `vw` for consistency's sake. There are (unfortunately) currently a lot of things that could break in the front-end, so there is an issue open for fixing this in general. Once we find a proper solution, we can apply it to all components.